.slashrc

```diff
@@ -122,6 +122,17 @@ class MediaPlugin(slash.plugins.PluginInterface):
       os.remove(tstfile)
     return tstfile
 
+  def _test_purge_artifact(self, filename):
+    tstfile = os.path.join(slash.context.result.get_log_dir(), filename)
```

Comment:
This function will take the absolute filename that was previously obtained from calling `get_media()._test_artifact`.  Thus, this `filename` should already contain the test log directory.  Instead, we should check if `filename` is in the `slash.context.result.data.get("artifacts", list())` (where it is stored by `get_media()._test_artifact`), and return immediately if it is not.  This will also prevent removing files that were not obtained by `get_media()._test_artifact()`.

Reply:
fixed as for review comment